### PR TITLE
fix: comments button color and type

### PR DESCRIPTION
**Pull request by nensidosari**
### Describe what this PR does
- removed secondary type from button to get the type based on figma
- on button hover the color of nr of comments should be same color as the highlighted comments button

Screenshot 2024-01-22 at 11 23 18 PM
Screenshot 2024-01-22 at 11 23 12 PM

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?
- [x] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices? No
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-116 #done

### CODE REVIEW

File: packages/shared/src/components/cards/v1/ActionButtons.tsx [hunk 1]
```diff
@@ -152,23 +152,17 @@ export default function ActionButtons({
           <Link href={post.commentsPermalink}>
             <Button
               id={`post-${post.id}-comment-btn`}
-              className="pointer-events-auto ml-2"
+              className="pointer-events-auto ml-2 hover:text-theme-color-blueCheese"
               color={ButtonColor.BlueCheese}
-              icon={<CommentIcon secondary={post.commented} />}
```
Comment (sshanzel) [inline packages/shared/src/components/cards/v1/ActionButtons.tsx:157]: I think this was intended right? When the user has made a comment we fill it.

Comment (nensidosari) [reply packages/shared/src/components/cards/v1/ActionButtons.tsx:157]: Ah yes, could be, it's not in designs, but if this is what the behaviour used to be, should be the same. 
Would `post.commented` check if the user itself has commented? Or if the post is commented by any user?

File: packages/shared/src/components/cards/v1/ActionButtons.tsx [hunk 2]
```diff
@@ -152,23 +152,17 @@ export default function ActionButtons({
...
+              icon={<CommentIcon />}
               tag="a"
               href={post.commentsPermalink}
-              pressed={post.commented}
               variant={ButtonVariant.Float}
               {...combinedClicks(() => onCommentClick?.(post))}
             >
               {post?.numComments > 0 ? (
                 <InteractionCounter
-                  className={classNames(
```
Comment (nensidosari) [inline packages/shared/src/components/cards/v1/ActionButtons.tsx:166]: I'll change this back as well based on the question/answer above

File: packages/shared/src/components/cards/v1/ActionButtons.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/cards/v1/ActionButtons.tsx:157]: Yeah, I checked the Figma and it doesn't seem to be there. It checks whether the logged in user has made a comment.

Comment (nensidosari) [reply packages/shared/src/components/cards/v1/ActionButtons.tsx:157]: Have updated to only make the required change. thank u!!

**Review by sshanzel — APPROVED** [review]
Looks great! 🚀

**Review by omBratteng — APPROVED** [review]

**Pull request merged**